cost/google/cud_report/README.md

```diff
@@ -0,0 +1,32 @@
+## Google Committed Use Discount (CUD) Report
+ 
+### What it does
+This policy identifies all CUDs that exist in a given GCP project and provides a report listing them all. It can optionally report on all CUDs, or only those that are active or expired.
+ 
+### Functional Details
+ 
+Uses the GCP API to get a list of all CUDs and report on them.
+ 
+#### Input Parameters
+ 
+- *CUD Status* - Allow the user to choose from "Active", "Expired", or "All"
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+ 
+### Required RightScale Roles
+ 
+- policy_manager
+ 
+### Supported Clouds
+ 
+- Google
+ 
+### Cost
+ 
+This Policy Template does not incur any cloud costs.
+
+### Prerequisite to apply this policy
+
+- Add New credentials (GC_SA_CLIENT_EMAIL and GC_SA_PRIVATE_KEY) *if doesnot exists* under Rightscale cloud managemant Desing/Credentials. 
```

Comment:
I don't see either of these credentials used in the policy.

What would the email cred be used for, if it was used?

Check spelling in this sentence.

Change Rightscale to RightScale

Reply:
I have renamed "GCE_PLUGIN_ACCOUNT and GCE_PLUGIN_PRIVATE_KEY" to "GC_SA_CLIENT_EMAIL and GC_SA_PRIVATE_KEY" respectively.

 'email cred' is a client email of the particular service account, which is used for authentication.